docs/examples/move/nft_marketplace/README.md

```diff
@@ -0,0 +1,119 @@
+# Marketplace Guide
+
+The `nft_marketplace.move` module provides a straightforward implementation of a marketplace extension. To utilize it, follow the steps outlined below.
+The `item_for_market.move` contains mocked item data for use within the marketplace.
+The `rental_extension.move` is an extension adds functionality to enable item rentals.
+
+
+## Steps to Use the Marketplace
+
+### 1. Connect to the Network
+Connect to the IOTA network (e.g., using a faucet to obtain tokens).
+
+### 2. Install Kiosk
+Run the following command to install the Kiosk module:
+
+```bash
+iota client call \                                   
+    --package 0x2 \
+    --module kiosk \
+    --function default
+```
+
+### 3. Publish `item_for_market.move`
+
+```bash
+iota client publish
+```
+
+### 4. Create an Item
+
+Create an item, for instance:
+
+```bash
+iota client call \
+    --package $M_ITEMS_ID \
+    --module market_items \
+    --function new_jeans
+```
+
+After creation, export the following variables:
+
```

Comment:
Add description for each variable to link with the corresponding returned value

Reply:
Done